phpunit.xml.dist

```diff
@@ -3,26 +3,23 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
     bootstrap="./vendor/autoload.php"
-    beStrictAboutCoversAnnotation="true"
-    beStrictAboutResourceUsageDuringSmallTests="true"
     beStrictAboutChangesToGlobalState="true"
+    beStrictAboutCoverageMetadata="true"
     beStrictAboutOutputDuringTests="true"
-    beStrictAboutTodoAnnotatedTests="true"
     executionOrder="random"
+    cacheDirectory=".phpunit.cache"
```

Comment:
Should be skipped

Reply:
Do we want to keep `.phpunit.result.cache` files?

Reply:
Nope, we don't really need any caching at all :thinking: